03-Raster-Calculations-In-R.Rmd

```diff
@@ -133,6 +145,23 @@ plot(DSM_HARV,
      main="Digital Surface Model (Elevation)\n Harvard Forest")
 ```
 
+Notice that the elevation range for these two maps (DSM and DTM) is different. 
+Why?  Should we be worried about this?  
+
```

Comment:
ok gotcha - you moved this down. if it makes more sense here, please ignore the previous comment.